### ISSUE TITLE
Strawberry doesn't output any audio

### ISSUE THREAD

**Issue by Czechball** [strawberrymusicplayer/strawberry#1425]
- [x] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [x] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [x] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
There is no audio output coming from Strawberry when playing any kind of audio file. There is no audio volume activity visible in PulseAudio Volume Control UI either.

**To Reproduce**
Open strawberry, play any audio file.

**Expected behavior**
I expect to hear audio.

**Screenshots:**
Not applicable

**System Information:**
 - Operating system: `6.7.12-1-MANJARO`
 - Strawberry Version: `1.0.23`

**Additional context**
It was working a few weeks ago without any issues. I haven't observed any particular change that might've caused this. I have tried rolling back to a previous kernel.

**Comment by jonaski** [comment]
Is this with pulseaudio, pipewire or directly to alsa? What's your output and device set to in backend settings? What's your audio device? Have you tried different audio devices?

**Comment by Czechball** [comment]
Hello,
yes, I was using Pulseaudio. I tried multiple devices, my default device is my laptop's 3.5mm jack (`00:1f.3 Multimedia audio controller: Intel Corporation Tiger Lake-LP Smart Sound Technology Audio Controller (rev 20)`)
However, it seemed to be an issue on my end. I tried downgrading to kernel `6.6.26-1-MANJARO` and got it working. I suspect there might've been an issue regarding the Intel audio drivers. However, I'm currently unable to reproduce the issue, so I'm closing this issue.
